index.html

```diff
@@ -58,11 +61,73 @@
       </div>
     </div>
   </nav>
+  <!--Our Customers-->
+  <section class="slider-1" id="slider-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <h4>
+            Trusted by over <span class="blue">1500</span> customers worldwide
```

Comment:
If you change the class name, don't forget to change it here as well.

```suggestion
            Trusted by over <span class="text-blue">1500</span> customers worldwide
```